tools/setup/emoji/generate_emoji_names

```diff
@@ -137,6 +137,7 @@ def main() -> None:
             emoji = "".join(
                 chr(int(h, 16))
                 for h in (emoji_dict["non_qualified"] or emoji_dict["unified"]).split("-")
+                if h != "FE0F"
```

Comment:
Can you add a comment linking to the upstream iamcal bug, to help remind us we can hopefully delete this once they fix it?